tests/unish/UnishIntegrationTestCase.php

```diff
@@ -102,7 +102,7 @@ public function drush($command, array $args = [], array $options = [], $expected
     protected function buildCommandLine($command, $args, $options)
     {
         $global_option_list = ['simulate', 'root', 'uri', 'include', 'config', 'alias-path', 'ssh-options', 'backend', 'cd'];
-        $options += ['root' => $this->webroot(), 'uri' => self::INTEGRATION_TEST_ENV]; // Default value.
+        $options += ['root' => $this->webroot(), 'uri' => self::INTEGRATION_TEST_ENV, 'strict' => 0]; // Default value.
```

Comment:
Why disable strict mode?

Reply:
Disabling strict mode bypasses argument validation. We can try without this.